cvmfs/catalog_sql.h

```diff
@@ -298,7 +298,7 @@ class SqlLookup : public SqlDirent {
                            const bool expand_symlink = true) const;
 
   /**
-   * DirectoryEntrys do not contain their path hash.
+   * DirectoryEntries do not contain their path hash.
```

Comment:
My tool wouldn't complain about `DirectoryEntry's`, `DirectoryEntry-s`, or `Directory Entries`...

But, I didn't see any sign of `DirectoryEntry` so...